examples/RestEaseClientGeneratorConsoleApp/Examples/Pitane/IPitaneApi.cs

```diff
@@ -0,0 +1,5046 @@
+        [Get("/pitaneSendCustomerAbsentSummaryByGUID")]
+        Task<object> GetPitaneSendCustomerAbsentSummaryByGUIDAsync([Query(Name = "rei_guid")] string reiGuid);
+
+        /// <summary>
+        /// GetPitaneCollectTrips (/pitaneCollectTrips)
+        /// </summary>
+        /// <param name="plaDatumtijdFrom">Start date/time collection.</param>
+        /// <param name="plaDatumtijdTo">End date/time collection.</param>
+        /// <param name="plaSoortvervoer">Travel kind collection</param>
+        /// <param name="plaRolstoel">Wheelchair only</param>
+        /// <param name="plaRolstoelExcluded">Wheelchair trips excluded</param>
+        /// <param name="plaEigenaarFiltered">Wheelchair trips excluded</param>
+        /// <param name="plaEigenaar">Trip owner (optional)</param>
+        [Get("/pitaneCollectTrips")]
+        Task<object> GetPitaneCollectTripsAsync([Query(Name = "pla_datumtijd_from")] string plaDatumtijdFrom, [Query(Name = "pla_datumtijd_to")] string plaDatumtijdTo, [Query(Name = "pla_soortvervoer")] string plaSoortvervoer, [Query(Name = "pla_rolstoel")] bool plaRolstoel, [Query(Name = "pla_rolstoel_excluded")] bool plaRolstoelExcluded, [Query(Name = "pla_eigenaar_filtered")] bool plaEigenaarFiltered, [Query(Name = "pla_eigenaar")] string plaEigenaar);
+
+        /// <summary>
+        /// GetPitaneCollectTripsByAPI (/pitaneCollectTripsByAPI)
+        /// </summary>
+        /// <param name="subToken">Start date/time collection.</param>
+        /// <param name="plaDatumtijdFrom">Start date/time collection.</param>
+        /// <param name="plaDatumtijdTo">End date/time collection.</param>
+        /// <param name="plaSoortvervoer">Travel kind collection</param>
+        /// <param name="plaRolstoel">Wheelchair only</param>
+        /// <param name="plaRolstoelExcluded">Wheelchair trips excluded</param>
+        /// <param name="plaEigenaarFiltered">Wheelchair trips excluded</param>
+        /// <param name="plaEigenaar">Trip owner (optional)</param>
+        [Get("/pitaneCollectTripsByAPI")]
+        Task<object> GetPitaneCollectTripsByAPIAsync([Query(Name = "sub_token")] string subToken, [Query(Name = "pla_datumtijd_from")] string plaDatumtijdFrom, [Query(Name = "pla_datumtijd_to")] string plaDatumtijdTo, [Query(Name = "pla_soortvervoer")] string plaSoortvervoer, [Query(Name = "pla_rolstoel")] bool plaRolstoel, [Query(Name = "pla_rolstoel_excluded")] bool plaRolstoelExcluded, [Query(Name = "pla_eigenaar_filtered")] bool plaEigenaarFiltered, [Query(Name = "pla_eigenaar")] string plaEigenaar);
+
+        /// <summary>
+        /// Create route
+        /// </summary>
+        /// <param name="content">Collection of combined trip records</param>
+        [Post("/pitaneUpdateTripCollection")]
+        [Header("Content-Type", "application/json")]
+        Task<object> PostPitaneUpdateTripCollectionAsync([Body] System.Collections.Generic.List`1[RestEaseClientGenerator.Models.Internal.PropertyDto][] content);
```

Comment:
Hmm, this looks weird to me. Do you have any idea where this comes from?

Reply:
This is a bug.

I'll check the code.